src/graphql/resolveConnection.js

```diff
@@ -8,6 +8,7 @@ const resolveConnection = (
   getFromClause = constant(table.getIdentifier()),
 ) => {
   const columns = table.getColumns()
+  const fullPrimaryKey = table.getPrimaryKeys()
```

Comment:
So I (or others) didn't accidentally use it instead the "no order by variant" 😊

Do you think that's an ok justification?

Also, for historical purposes (I should probably doc this inline) the reason I filtered out `orderBy` was because I was hitting really wired bugs where `(id, id) > (1, 1)` would *include* rows with an `id` of 1 instead of excluding them as the `>` operator assumes. This was happening locally in my PG 9.5 (beta?) instance, but not in the CI 9.4 instance. In any case though I think it's useful to be deduping that comparison of possible.